### PR TITLE
doc(Anchor): Added description

### PR DESCRIPTION
**Pull request by jbwriter**

### CODE REVIEW

File: apps/website/src/pages/docs/anchor.mdx [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 title: Anchor
-description:
+description: The anchor component works like a simple hyperlink, targeting a specific URL. Additionally, it can also target a named location within a page. For example, you might want to target a content area that appears below the fold so the user does not have to scroll to discover it.
```
Comment (mayank99) [inline apps/website/src/pages/docs/anchor.mdx:3]: This is a bit too wordy. Descriptions need to be short enough to fit in 3 small lines

**Comment by elephantcatdog** [comment]
@elephantcatdog Update text to "an anchor"

File: apps/website/src/pages/docs/anchor.mdx (hunk 1 above)
Comment (jbwriter) [reply apps/website/src/pages/docs/anchor.mdx:3]: Valid. Didn't realize that the "Description" field is used in the tile synopsis. There are plenty reasons to provide more clarifying text (I think "wordy" might be a bit harsh ;) ), but I will include extra information about the component below the first live example and not in the "Description" field. First timer here.

File: apps/website/src/pages/docs/anchor.mdx [hunk 2]
```diff
@@ -15,6 +15,8 @@ import LiveExample from '~/components/LiveExample.astro';
 
 <LiveExample src={frontmatter.exampleCodeFile1} />
 
+Consider using a named anchor when you want the user to land not only on a specific page, but to actually arrive at a content area that may be below the fold so the user does not have to scroll to discover it.
```
Comment (mayank99) [inline apps/website/src/pages/docs/anchor.mdx:18]:
```suggestion
Consider using a named anchor when you want the user to land not only on a specific page, but also to arrive at a content area that may be below the fold so the user does not have to scroll to discover it.
```

File: apps/website/src/pages/docs/anchor.mdx [hunk 3]
```diff
@@ -1,6 +1,6 @@
 ---
 title: Anchor
-description:
+description: The anchor component works like a simple hyperlink, targeting a specific URL or named anchor within a page.
```
Comment (mayank99) [inline apps/website/src/pages/docs/anchor.mdx:3]:
```suggestion
description: The anchor component creates a hyperlink targeting a specific location within the same page or a different one.
```

File: apps/website/src/pages/docs/anchor.mdx (hunk 1 above)
Comment (mayank99) [reply apps/website/src/pages/docs/anchor.mdx]: Definitely, there is a lot to document even for a very simple component like this one (for instance, anchors can link to not just pages but also to files). If you would like to further expand on the description, feel free to do so before the paragraph you just added.
